use-cases.html

```diff
@@ -0,0 +1,303 @@
+        </li>
+      </ul>
+      <p>
+        Core constraints when obtaining the gelocation are <strong>accuracy</strong>
+        (<em>how close to the actual position of the user is the determined position</em>) and
+        <strong>latency</strong> (<em>how long does the user want to wait for a result</em>).
+        Both are tradeoffs: one can trade faster results for lower accuracy and vice versa.
+      </p>
+      <p>
+        A common theme is to first obtain a rough estimation that then gets refined over time,
+        for example based initially on surrounding WiFi signals (which is fast to obtain)
+        and then eventually based on precise GPS data (which may take some time to find a signal).
+      </p>
+    </section>
+
+    <section id="use_cases" class="informative">
+      <h2>Use Cases</h2>
+      <p>
+        In the following, we list use cases based on the previously defined categories.
+        We note that the categories are not mutually exclusive and that overlaps exist.
+        A task might start in the foreground, then continue in the background
+        (for example, while the user quickly responds to an incoming email), and then eventually
+        terminate in the foreground when the user multitasks back.
+      </p>
+
+      <section id="usecases_oneoff_foreground">
+        <h3>Foreground—One-off geolocation update</h3>
+
+        <section id="usecases_whereami">
+          <h4>Locate a user on a map</h4>
+          <p>
+            A mapping application can use the Geolocation Sensor API data of a user
+            to locate them on the map, essentially responding to the question
+            "Where am I right now?"
+          </p>
+        </section>
+
+        <section id="usecase_find_poi">
+          <h4>Find points of interest in the user's area</h4>
+          <p>
+            Someone visiting a foreign city could access a web application that
+            allows users to search or browse through a database of tourist
+            attractions. Using the Geolocation Sensor API, the web application has access to
+            the user's approximate current position and is therefore able to rank the
+            search results by proximity to the user's location.
+          </p>
+        </section>
+
+      </section>
+
+      <section id="usecases_continuous_foreground">
+        <h3>Foreground—Continuous geolocation updates</h3>
+
+        <section id="usecase_local_info">
+          <h4>Up-to-date local information</h4>
+
+          <p>
+            A widget-like web application that shows the weather or news that are
+            relevant to the user's current area can use the Geolocation Sensor API to
+            register for location updates. If the user's position changes, the widget
+            can adapt the content accordingly.
+          </p>
+        </section>
+
+        <section id="usecase_poi_alert">
+          <h4>Alerts when points of interest are in the user's vicinity</h4>
```

Comment:
This feels more like geofencing, otherwise the user would have to be actively using the application before the notification is generated.

Reply:
@reillyeon The use case was indeed written with an actively interested person in mind who would have a tourist guide app open in the foreground.

Reply:
Can you clarify that in the description?

Reply:
Will do.